.github/workflows/ci.yml

```diff
@@ -40,3 +40,30 @@ jobs:
       - run: script/clone_all_rspec_repos
       - run: bundle install --binstubs --standalone
       - run: script/run_build
+
+  windows:
+    name: Ruby ${{ matrix.ruby }} (Windows)
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        ruby:
+          - 2.7
+          - 2.6
+          - 2.5
+          - 2.4
+          - 2.3
+          - 2.2
+          - 2.1.9
+      fail-fast: false
+    continue-on-error: ${{ matrix.ruby == '2.7' }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          bundler: 2
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
+      - run: cinst ansicon
+      - run: bundle exec rspec --backtrace
+        env:
+          APPVEYOR: true
```

Comment:
Is is still needed?

Reply:
It was until support was merged changing the checks.